disassemble_f1.py

```diff
@@ -172,7 +116,8 @@
         add_string(memory, addr)
 
 
-    apply_comments(comments)
+    mem = MemoryWrapper(memory)
+    read_annotations("roms/Diag_F1_Rev_1.0.comments", mem)
```

Comment:
Missing file?

Reply:
Ah, yes, it's my disease - forgetting git add